cob_bringup_sim/scripts/spawn_model.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/python
```

Comment:
what does spawn_model do different than spawn_object? Can't this be integrated into one spawn script?